MatrixSDK/NotificationCenter/MXNotificationCenter.m

```diff
@@ -450,12 +456,21 @@ - (void)updatePushRuleActions:(NSString*)ruleId
         // Refresh locally rules
         [self refreshRules:^{
             [[NSNotificationCenter defaultCenter] postNotificationName:kMXNotificationCenterDidUpdateRules object:self userInfo:nil];
+            if (completion) {
+                completion(nil);
+            }
         } failure:^(NSError *error) {
             [[NSNotificationCenter defaultCenter] postNotificationName:kMXNotificationCenterDidFailRulesUpdate object:self userInfo:@{kMXNotificationCenterErrorKey:error}];
+            if (completion) {
+                completion(nil);
+            }
         }];
     }
                                                  failure:^(NSError *error) {
```

Comment:
Let's maybe move this failure to the previous line

Reply:
Can you clarify this suggestion?
Rationale so far: call the completion block **without** an error after `refreshRules` completes (even if there's an error on that).
This is because the error I'm caring of here is related to the update of the `actions` and not to the get of the push rules.

Reply:
Ah no sorry, I was just talking about alignment:
![Screenshot 2023-01-31 at 10 28 59](https://user-images.githubusercontent.com/637564/215708147-7c74ab4b-a721-4cf7-9a87-5f9c313dfae8.png)

vs. 

![Screenshot 2023-01-31 at 10 29 04](https://user-images.githubusercontent.com/637564/215708152-62b003bd-086e-4933-aaa2-7791232f79d2.png)

Reply:
The alignment looks ok to me.
This is what I get doing ctrl+I on the full method (as it's now).

![align](https://user-images.githubusercontent.com/19324622/215709336-a8af70e8-a4aa-4516-a094-9be3cbffc33d.png)

wdyt?

Reply:
I just find it weird because that failure block belongs to the parent method but it's indented way deep into the hierarchy, making it hard to parse. I much rather prefer this version:

![Screenshot 2023-01-31 at 11 18 31](https://user-images.githubusercontent.com/637564/215719296-928880b4-82bf-4091-a880-f0e84519a686.png)

But in the grand scheme of things, this doesn't really matter. I'll leave it completely up to you 👍

Reply:
Applied ;)
https://github.com/matrix-org/matrix-ios-sdk/pull/1702/commits/0fe0d740256f71793e44c3fa04ba0a7f9638543c